.circleci/config.yml

```diff
@@ -42,7 +42,12 @@ jobs:
       NODE_DEBUG: gh-pages
     steps:
       - *attach_workspace
-      - run: npm run build:demo
+      - run: |
+          if [ "$CIRCLE_BRANCH" == "main" ]; then
+            npx @eslint/config-inspector build --base eslint-config
```

Comment:
Sorry, should've described .. it's buried in https://github.com/eslint/config-inspector/blob/62cd3990a9d7b32316e6c89bdc6aa00470e3b8a4/src/cli.ts#L25 and fixes the first bullet of the PR description

> [...] absolute URLs that attempt to resolve against https://zendeskgarden.github.io/ instead of https://zendeskgarden.github.io/eslint-config/

Reply:
Awesome!